components/AppMenu.jsx

```diff
@@ -173,9 +176,13 @@ class AppMenu extends React.Component {
             document.addEventListener('keydown', this.onKeyPress, true);
             document.addEventListener('mousemove', this.onMouseMove, true);
         } else {
-            document.removeEventListener('click', this.checkCloseMenu);
-            document.removeEventListener('keydown', this.onKeyPress, true);
-            document.removeEventListener('mousemove', this.onMouseMove, true);
+            if (this.props.keepMenuOpen) {
+                return;
+            } else  {
+                document.removeEventListener('click', this.checkCloseMenu);
+                document.removeEventListener('keydown', this.onKeyPress, true);
+                document.removeEventListener('mousemove', this.onMouseMove, true);
```

Comment:
I just debugged an issue related to these event listeners. If they remain active, they will break arrow-pan of the map and i.e. enter-submit of input fields. So it is undesirable to have these event listeners active unless there really is an open menu.

Reply:
Ok I will remove these changes